ci/gpu/build.sh

```diff
@@ -53,7 +53,7 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=${MINOR_VERSION}" \
-      "xgboost=1.2.0dev.rapidsai${MINOR_VERSION}" \
+      "xgboost=1.3.0dev.rapidsai${MINOR_VERSION}" \
```

Comment:
Is the Conda package `xgboost=1.3.0dev.rapidsai` already available?

Reply:
yup, fresh off the conda presses here: https://anaconda.org/rapidsai-nightly/xgboost/files?version=1.3.0dev.rapidsai0.17